packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx

```diff
@@ -33,7 +33,11 @@ function GoogleMapsMap({ gmApiKey, onMapView, onPositionControl, mapsIndoorsInst
             // Initialize Google Maps MapView
             const mapViewOptions = {
                 element: document.getElementById('map'),
-                disableDefaultUI: true // Disable Map Type control, Street view control and Zoom controls.
+                disableDefaultUI: true, // Disable Map Type control, Street view control and Zoom controls.
+                // Always set a center so the map so the bounds or center can be read from the start.
+                center: { lat: 0, lng: 0 },
+                // Set a large zoom so we prevent a "zoom 0 glitch" (showing the whole globe temporarily)
+                zoom: 21
```

Comment:
How does that work with the startZoomLevel query param on Google Maps?

Reply:
That is set at a later time and should not clash. Just like fitting to the venue will implicitly change the zoom level.